ouroboros-consensus/src/Ouroboros/Consensus/Mempool/TxLimits.hs

```diff
@@ -31,20 +47,58 @@ import           Ouroboros.Consensus.Ticked (Ticked (..))
 -- eras (starting with Alonzo) this measure was a bit more complex
 -- as it had to take other factors into account (like execution units).
 -- For details please see the individual instances for the TxLimits.
-class (Monoid (Measure blk)) => TxLimits blk where
+class BoundedMeasure (TxMeasure blk) => TxLimits blk where
+  type TxMeasure blk
 
-  type Measure blk
+  -- | What is the measure an individual tx?
+  txMeasure        :: Validated (GenTx blk)    -> TxMeasure blk
 
-  lessEq       :: Measure blk -> Measure blk -> Bool
-  txMeasure    :: Validated (GenTx blk) -> Measure blk
-  maxCapacity  :: Ticked (LedgerState blk) -> Measure blk
-  pointwiseMin :: Measure blk -> Measure blk -> Measure blk
+  -- | What is the allowed capacity for txs in an individual block?
+  txsBlockCapacity :: Ticked (LedgerState blk) -> TxMeasure blk
+
+-- | Is every component of the first value less-than-or-equal-to the
+-- corresponding component of the second value?
+(<=) :: Measure a => a -> a -> Bool
+(<=) = (Measure.<=)
+
+{-------------------------------------------------------------------------------
+  ByteSize
+-------------------------------------------------------------------------------}
 
 newtype ByteSize = ByteSize { unByteSize :: Word32 }
-  deriving stock (Show, Eq, Ord)
+  deriving stock (Show)
+  deriving newtype (Eq, Ord)
+  deriving newtype (BoundedMeasure, Measure)
+
+{-------------------------------------------------------------------------------
+  Overrides
+-------------------------------------------------------------------------------}
+
+-- | How to override the limits set by the ledger state
+--
+-- The forge logic must use the @Measure.min@ of the limits from ledger state
+-- and this override. More concretely: for each component of @'TxMeasure' blk@
+-- it will use the minimum of the ledger's limits and this override.
+newtype Overrides blk = Overrides {getOverrides :: TxMeasure blk}
```

Comment:
Elaborate here: motivate the newtype

  - specifically what kind of `TxMeasure` is this?
  
  - allow a hook for the `Semigroup` and `Monoid` instances to mean conjunction (ie `meet`) instead of addition.
  
Call out the 'Measure' semantics and specifically the lattice-i-ness of it.